csrankings-m.csv

```diff
@@ -1162,6 +1162,7 @@ Matei Ripeanu,University of British Columbia,http://www.ece.ubc.ca/~matei,eeMpO3
 Matei Zaharia,Univ. of California - Berkeley,https://people.eecs.berkeley.edu/~matei/,I1EvjZsAAAAJ
 Matej Hoffmann,Czech Technical University,https://sites.google.com/site/matejhof,KFUbXYYAAAAJ
 Mateja Jamnik,University of Cambridge,https://www.cl.cam.ac.uk/~mj201,NOSCHOLARPAGE
+Mateo Díaz,Johns Hopkins University,https://mateodd25.github.io/,F3cPGhsAAAAJ

```

Comment:
<img width="1224" alt="Screen Shot 2024-10-26 at 11 15 10 AM" src="https://github.com/user-attachments/assets/49547e0f-f69a-4c59-8833-c9ad4bf9374f">

https://www.cs.jhu.edu/people/cs-affiliates/joint-and-secondary-faculty/